### DISCUSSION TITLE
Is it possible to adjust the spacing between the filenames?

### DISCUSSION THREAD

**Discussion by Piero24** [eza-community/eza#963] [Q&A, Howto, Help]
Hi, Is it possible to change the space between the filenames? I have checked the manual, but I have only found the possibility of increasing the space between the icons and the filenames, not the possibility of increasing the space between each file.

### Here is an example:

This is the output from `eza`:

![1](https://github.com/eza-community/eza/assets/32714047/a8404a51-1785-42f4-98ef-c93da5cb6350)

This one (from standard `ls`) is the output that I'm looking for (**Note** that the spaces are not equally distributed. If they were equally distributed, it would be better):

![2](https://github.com/eza-community/eza/assets/32714047/3f33737f-e699-4150-8907-fa3a6d7a32f9)

**Comment by gierens** [comment]
You can likely pipe this into some other tool to reorganize the columns, like `column` for example.

`eza` itself doesn't allow this at the moment. The logic creating the grid is fairly complex and we have to rewrite it at some point and then could probably add this feature, but that's nothing that is going to happen very soon.

Feel free to open a feature request though so it stays on our radar.
